modules/ROOT/pages/sap/sap-connector.adoc

```diff
@@ -269,19 +272,38 @@ There are no special Advanced properties for this configuration.
 |Log trace flag |logTraceFlag |Indicates if the trace should be logged on the Mule server.|false
 |===
 
-== Use Case: Send an IDoc to SAP
+== Use Case: Function Source - Return Values to SAP
+
+The current use case describes how to create a Mule application that receives a function using *Function source* which passes the export parameter back to SAP.
```

Comment:
the function passes the export parameter back to SAP, or the app? Rewrite for clarity? (I'm guessing that lack of comma before "which" should tell me it's the app, but it would be better to be clear). Maybe move the "using" clause to end of sentence, or...?